models/app.models.js

```diff
@@ -26,3 +25,12 @@ exports.selectArticlesById = (article_id) => {
     return rows[0];
   });
 };
+
+exports.selectCommentsByArticleId = (article_id) => {
```

Comment:
great start with this endpoint - once you incorporate the tests mentioned above your model might change :)